.github/workflows/precommit.yml

```diff
@@ -411,6 +411,63 @@ jobs:
             .tox/.coverage
             .tox/coverage.xml
 
+  test-modin-extra-without-pandas-extra:
```

Comment:
Can this be in daily precommit tests?

Reply:
Can you move to the daily modin precommit once https://github.com/snowflakedb/snowpark-python/pull/1421/files is in?

Reply:
Why should we move this to daily? It's just running a single test case, so it's fast.

Reply:
+1, I think we can keep this in

Reply:
sgtm if it is super fast.